packages/rebber/src/types/table.js

```diff
@@ -0,0 +1,31 @@
+/* Expose. */
+const one = require('../one')
+module.exports = table
+
+const defaultMacro = (ctx, node) => {
+  let j = 0
+  const parsed = []
+  node.children.map(n => parsed.push(one(ctx, n, j++, node)))
```

Comment:
in fact I tried this but it didn't work. However in between i made mor changements so perhaps I can rollback to this.